### PR TITLE
VDS-1196: Move simple export to simple-export-import module

### PR DESCRIPTION
**Pull request by ilyawzrd**

### CODE REVIEW

File: src/VirtoCommerce.SimpleExportImportModule.Core/Models/SimpleExportTabularPrice.cs [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+{
+    public class SimpleExportTabularPrice : TabularPrice
+    {
+        public string Code { get; set; }
+        public DateTime? ModifiedDate { get; set; }
+        public DateTime? StartDate { get; set; }
+        public DateTime? EndDate { get; set; }
+        public DateTime CreatedDate { get; set; }
+        public string CreatedBy { get; set; }
+        public string ModifiedBy { get; set; }
```
Comment (asvishnyakov) [inline src/VirtoCommerce.SimpleExportImportModule.Core/Models/SimpleExportTabularPrice.cs:14]: May we add blank lines between?

File: src/VirtoCommerce.SimpleExportImportModule.Core/ModuleConstants.cs [hunk 2]
```diff
@@ -17,5 +20,31 @@ public static class Permissions
...
+                };
+
+                public static IEnumerable<SettingDescriptor> AllSettings
+                {
+                    get
+                    {
+                        return new List<SettingDescriptor>
+                               {
+                                   SimpleExportLimitOfLines
+                               };
```
Comment (asvishnyakov) [inline src/VirtoCommerce.SimpleExportImportModule.Core/ModuleConstants.cs:74]: Fix ident

**Review by asvishnyakov — CHANGES_REQUESTED** [review]

File: src/VirtoCommerce.SimpleExportImportModule.Core/Models/SimpleExportTabularPrice.cs (hunk 1 above)
Comment (ilyawzrd) [reply src/VirtoCommerce.SimpleExportImportModule.Core/Models/SimpleExportTabularPrice.cs:14]: done

File: src/VirtoCommerce.SimpleExportImportModule.Core/ModuleConstants.cs (hunk 2 above)
Comment (ilyawzrd) [reply src/VirtoCommerce.SimpleExportImportModule.Core/ModuleConstants.cs:74]: discussed, wont fix

**Review by asvishnyakov — APPROVED** [review]

**Pull request merged**